### PR TITLE
Collections: HashTable, LinkedList and Queue

### PR DESCRIPTION
**Pull request by galvisma**
This PR contains examples of the basic instructions to create, add, delete and display the content of a Hashtable, LinkedList and Queue.

### CODE REVIEW

File: src/hashTable/Main.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    private static void readingTablaHash(Hashtable<Integer, String> table) {
+        Enumeration<String> enumerationNames = table.elements();
+        Enumeration<Integer> enumerationKeys = table.keys();
+        while (enumerationNames.hasMoreElements()) {
+            System.out.println("key: " + enumerationKeys.nextElement() + " " +
+                    "Name: " + enumerationNames.nextElement());
+        }
+    }
```
Comment (julianespinel) [inline src/hashTable/Main.java:53]: This is correct, however, there is an easier way to do this:

```java
    private static void readingTablaHash(Hashtable<Integer, String> table) {
        Set<Integer> keys = table.keySet();
        for (Integer key : keys) {
            System.out.println("key: " + key + " " +"Name: " + table.get(key));
        }
    }
```

File: src/queueList/Main.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                        break;
+                    default:
+                        JOptionPane.showMessageDialog(null, "Invalid Option");
+                        break;
+                }
+            } catch (NumberFormatException e) {
+            }
+        } while (option != 4);
+    }
+}
```
Comment (julianespinel) [inline src/queueList/Main.java:45]: Please avoid using UI components in these exercises. Use the terminal instead.

In this queue exercise, the idea is to use the methods `add` and `remove` to see how a queue is a FIFO (First In First Out) data structure.

See:
* https://en.wikipedia.org/wiki/Queue_(abstract_data_type)
* https://docs.oracle.com/javase/8/docs/api/java/util/Queue.html

File: src/queueList/Queue.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        for (int i = string.length - 1; i >= 0; i--) {
+            invertedQueue += " " + string[i];
+        }
+        System.out.println("Content of Queue: " + invertedQueue);
+
+
+        JOptionPane.showMessageDialog(null, invertedQueue);
+        queue = "";
+    }
+}
```
Comment (julianespinel) [inline src/queueList/Queue.java:74]: Please avoid using UI components in these exercises. Use the terminal instead.

For this task, you don't have to create the `Queue` yourself.
The idea is to use the class Queue from Java. See: https://docs.oracle.com/javase/8/docs/api/java/util/Queue.html

**Review by julianespinel — CHANGES_REQUESTED** [review]
A few comments.

Please remove all the files `*.class`, and add the folder `out/` to the file `.gitignore` to avoid uploading these files in the future.

**Comment by galvisma** [comment]
This PR is closed in favor of last version collections

**Pull request closed without merging**